gpbft/gpbft.go

```diff
@@ -400,7 +405,7 @@ func (i *instance) validateMessage(msg *GMessage) error {
 	}
 
 	// Check vote signature.
-	sigPayload := msg.Vote.MarshalForSigning(i.participant.host.NetworkName())
+	sigPayload := i.participant.host.MarshalPayloadForSigning(&msg.Vote)
```

Comment:
This turned out to be "not terrible" because we already needed the host for the network name.